src/modules/jcms_migrate/migration_assets/subjects.json

```diff
@@ -97,6 +97,13 @@
     "image": "subjects/elife-sciences-biophysics-and-structural-biology-illustration.jpg",
     "alt": "Biophysics and Structural Biology"
   },
+  {
+    "id": "biochemistry-chemical-biology",
```

Comment:
I would prefer if we use migrate to create the subject in production. At least until I refactor how we ensure they are there on a clean install.

Reply:
Preferring that to creating it manually from the UI? I second this opinion.

Reply:
> Preferring that to creating it manually from the UI?

Yes